pages/api/level/[id].ts

```diff
@@ -83,6 +98,16 @@ export default withAuth(async (req: NextApiRequestWithAuth, res: NextApiResponse
     const stats = await StatModel.find<Stat>({ levelId: id });
     const userIds = stats.filter(stat => stat.complete).map(stat => stat.userId);
 
+    const worldsWithThisLevel = await WorldModel.findOne({levels: level._id})
```

Comment:
need a condition here in case no such world exists (deleting a level created before this pr should return undefined here)

also don't need it yet, but to prepare for multiple worlds we could change to something like:
```
const worldsWithThisLevel = await WorldModel.find(...)
const worldIds = worldsWithThisLevel.map(world => world._id)
WorldModel.updateMany({ _id: { $in: worldIds }}...
```

Reply:
Added